packages/view-types/feature-list/src/FeatureListOptions.js

```diff
@@ -35,62 +35,69 @@ export default function FeatureListOptions(props) {
     <OptionsContainer>
       {children}
       <TableRow>
-        <TableCell className={classes.labelCell} htmlFor="feature-list-sort-option-select">
+        <TableCell className={classes.labelCell} variant="head" scope="row">
           Sort Ordering
         </TableCell>
-        <TableCell>
+        <TableCell variant="body">
           <OptionSelect
+            key="feature-list-sort-option-select"
             className={classes.select}
             value={featureListSort}
             onChange={handleFeatureListSortChange}
             inputProps={{
-              id: 'feature-list-sort-option-select',
+              id: 'feature-list-sort-option',
```

Comment:
Hi @ivababukova ,

In general, whenever there is a component with this structure:

```
<TableRow>
  <TableCell>
     Some Label Text
  </TableCell>
  <TableCell>
    <SomeInput />
  </TableCell>
</TableRow>
```
Please do the following (if the MUI component supports `inputProps` then use that, otherwise supply `id={someId}` as a direct prop):


```
import { useId } from 'react';

// ...
const someId = useId();
// ...
<TableRow>
  <TableCell variant="head" scope="row">
     <label htmlFor={someId}>Some Label Text</label>
  </TableCell>
  <TableCell variant="body">
    <SomeInput inputProps={{ id: someId }} />
  </TableCell>
</TableRow>
```

(I am omitting extra props like `className`s here but please keep those)

For background, please see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes

Reply:
It is very strange, because I was sure I actually did that. I don't know what happened but my changes must have got lost. I have fixed all of that now. I also had to update the eslint rules in relation to this.